app/models/book.rb

```diff
@@ -0,0 +1,6 @@
+#  Modelo para el libro
+class Book < ApplicationRecord::Base
+  has_many :comentary
```

Comment:
commentaries, the names should reflect cardinality, if it's a has_many relationship, the name should be in plural, if it's a belongs_to or has_one, it should be in singular